src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1747,11 +1748,43 @@ const std::vector<tensor> inputShapes = {
     {16, 16, 5, 5},
 };
 
-const auto fpFunctionsParams = ::testing::Combine(::testing::ValuesIn(dataTypes),
-                                                  ::testing::ValuesIn(types),
-                                                  ::testing::ValuesIn(inputShapes),
-                                                  ::testing::ValuesIn(activationFunctions),
-                                                  ::testing::Values(activation_additional_params{}),
-                                                  ::testing::Values(padding{}));
-
-INSTANTIATE_TEST_SUITE_P(fp_activation_blocked_tests, activation_random_test, fpFunctionsParams);
+INSTANTIATE_TEST_SUITE_P(
+    fp_activation_blocked_tests0,
+    activation_random_test,
+    ::testing::Combine(::testing::Values(dataTypes[0]),
+                       ::testing::Values(types[0]),
+                       ::testing::Values(inputShapes[0]),
+                       ::testing::ValuesIn(std::vector<activation_func>(activationFunctions.begin() + activationFunctions.size()/4*0,
+                                                                        activationFunctions.begin() + activationFunctions.size()/4*1)),
```

Comment:
This division seems a bit weird... I think it would be better to test for whole activationFunctions.

Reply:
The number of testcase will be generated is 38*9=342 if we test whole activation function and whole format simultaneously. Is it ok?